doc/manuals/Substitute.am

```diff
@@ -17,6 +17,7 @@ SUBSTITUTE=sed "\
 	s%@SYSCONFDIR@%$(sysconfdir)%g;\
 	"
 
-## Example of how to substitute:
-## squid.8: $(srcdir)/squid.8.in Makefile
-##	$(SUBSTITUTE) < $(srcdir)/squid.8.in > $@
+.md.8:
+	$(SUBSTITUTE) < $(srcdir)/`basename $<` | \
```

Comment:
`.8` target (in this example and the actual use case) used to depend on Makefile. This change removes that dependency. Was that dependency wrong or unnecessary?